### PR TITLE
move deployment infrastructure from Core to Farajaland

### PR DESCRIPTION
**Pull request by naftis**
#3916

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -45,7 +47,7 @@ jobs:
         run: |
           cd ${{ github.event.repository.name }}
           echo "COUNTRY_CONFIG_VERSION=${{ github.event.inputs.countryconfig-image-tag }}" >> $GITHUB_ENV
-          git checkout ${{ github.event.inputs.countryconfig-image-tag }}
+          git checkout move-deployment-to-farajaland
```
Comment (naftis) [inline .github/workflows/deploy.yml]: remember to revert this

File: infrastructure/deploy.sh [hunk 2]
```diff
@@ -269,6 +269,18 @@ mkdir -p /tmp/opencrvs/infrastructure/hearth-plugins
...
+
+rotate_authorized_keys() {
+  # file exists and has a size of more than 0 bytes
+  if [ -s "/tmp/opencrvs/infrastructure/authorized_keys" ]; then
+    ssh $SSH_USER@$SSH_HOST 'cat /tmp/opencrvs/infrastructure/authorized_keys > ~/.ssh/authorized_keys'
+  else
+    echo "File /tmp/opencrvs/infrastructure/authorized_keys is empty. Did not rotate authorized keys!"
+  fi
+}
+
```
Comment (Zangetsu101) [inline infrastructure/deploy.sh]: not sure if this can be called rotating as we are just copying the contents of the file? :sweat_smile:

**Review by Zangetsu101 — APPROVED** [review]

File: infrastructure/deploy.sh (hunk 2 above)
Comment (naftis) [reply infrastructure/deploy.sh]: well we are overriding it every time with the latest values. Don't know if rotating is a good word. Maybe just override_authorized_keys

Comment (naftis) [reply infrastructure/deploy.sh]: Reverted the changes. Lets discuss further before committing to an approach

File: infrastructure/clear-all-data.sh [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+#---------------------------
+drop_database hearth-dev;
+
+drop_database openhim-dev
+
+drop_database user-mgnt;
+
+drop_database application-config
+
+drop_database metrics
```
Comment (Zangetsu101) [inline infrastructure/clear-all-data.sh]: we need to add the `performance` db here

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: infrastructure/clear-all-data.sh (hunk 3 above)
Comment (naftis) [reply infrastructure/clear-all-data.sh]: Done.

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**